tools/docs/main.go

```diff
@@ -42,6 +43,7 @@ func main() {
 
 	atlasBuilder := root.Builder()
 	atlasBuilder.InitDefaultCompletionCmd()
+	atlasBuilder.AddCommand(plugin.Builder(nil, nil))
```

Comment:
Why did you remove this function from the root builder?

Reply:
Calling the RegisterPlugin function in the `root.Builder()` will cause the unit tests for the builder to fail if there are any plugins installed because that will change the amount of sub commands of the rootCmd.